docs/prod-auth-first-login.md

```diff
@@ -7,18 +7,93 @@ section](prod-health-check-debug.html).)
 
 Once you've finished installing Zulip, configuring your settings.py
 file, and initializing the database, it's time to login to your new
-installation.  By default, initialize-database creates 1 realm that
-you can join, the `ADMIN_DOMAIN` realm (defined in
-`/etc/zulip/settings.py`).
+installation.
+
+## Create your user
+
+There are two ways to create your first user account: by registering through
+the Zulip web app or with the manage.py command.
+
+### Option 1: Register through Zulip web app
+
+Visit your Zulip instance in your web browser. You should see something like:
+
+![Image of Zulip home page](images/zulip-home.png)
+
+Click **Register** and provide your email address. You will be emailed a
+confirmation link. Follow that link and you'll be prompted to complete your
+registration by providing your full name and a password:
+
+![Image of Zulip create user page](images/zulip-confirm-create-user.png)
+
+### Option 2: User manage.py command
+
+The Zulip management console offers a number of useful commands. One of them is
+the ability to create users.
+
+You always need to run `manage.py` as the Zulip user. To start an interactive
+shell as the Zulip user, use `sudo -u zulip -i`.
+
+Use `manage.py` as follows to create your user, replacing ADMIN_DOMAIN with the
+domain (realm) that you specified in `/etc/zulip/settings.py`:
+
+```
+cd deployments/current
+/manage.py create_user --domain=ADMIN_DOMAIN --this-user-has-accepted-the-tos
+```
+
+You will be prompted to provide an Email and Full name. Your password will be
+set to a default one, but not revealed to you. If you would like to log in to
+Zulip using your email address and a password, you'll have to go through the
+reset password process. Otherwise, you can login with Google or another
+authentication method.
+
+## Grant administrator access
+
+You will likely want to make your own user account an admin user,
+which you can do via the following with the `knight` management command.
+
+You always need to run `manage.py` as the Zulip user. To start an interactive
+shell as the Zulip user, use `sudo -u zulip -i`).
+
+Be sure to replace `username@example.com` with the email address you used to
+create your Zulip user:
+
+```
+./manage.py knight username@example.com -f
+```
+
+Now that you are an administrator, you will have a special
+"Administration" tab linked to from the upper-right gear menu in the
+Zulip app that lets you deactivate other users, manage streams, change
+the Realm settings, etc.
+
+![Image of Zulip admin settings page](images/zulip-admin-settings.png)
+
+### Creating api super users with manage.py
+
+You can also use `./manage.py knight` with the
```

Comment:
Consider rewording this so that most people can ignore this after reading the first few words:  "If you need to manage IRC/Jabber/Zephyr mirrors, you will need to create super users.  To do this..."